docs/source/tutorials/Start A New Project.md

```diff
@@ -0,0 +1,264 @@
+# Start A New Project
+
+If developers want to start a new project(Such as paper reproduction and finetune task), how can we use Libai to achieve the least code work.
+
+The advantages of using Libai to develop a task are as follows:
+- Avoid repeating a lot of work, many functions of Libai can be directly inherited in projects.
+- We can quickly reproduce the experiments we have run, because Libai automatically saves the configuration file.
+- Various information will be output during training, such as remaining training time, current iter, throughput, loss information and current learning rate.
+- Only a few parameters need to be set for distributed training.
+
+
+## Introduce
+Let's use Bert_Finetune task as an example to introduce Libai, The complete project is [here](https://github.com/Oneflow-Inc/libai/tree/main/projects/QQP).
+
+For start a new Libai project, our main work is as follows:
```

Comment:
Main steps for starting a new Libai project:

Reply:
ok